packages/edit-site/src/components/editor/utils.js

```diff
@@ -53,33 +60,42 @@ export function useEditorFeature( featurePath, blockName = GLOBAL_CONTEXT ) {
 	);
 }
 
-export function getPresetVariable( presetCategory, presets, value ) {
+export function getPresetVariable( styles, blockName, propertyName, value ) {
 	if ( ! value ) {
 		return;
```

Comment:
Can we make it so this function returns the preset variable or the input value? I've noticed in all the places this is used we need a fallback for when we return undefined, which won't be needed if we return the input value when there's no need for processing it.

Reply:
Same for `getValueFromVariable`.

Reply:
Feedback applied 👍